report/distributionList.go

```diff
@@ -0,0 +1,52 @@
+package report
+
+import (
+	"io/ioutil"
+
+	"github.com/davecgh/go-spew/spew"
+	"gopkg.in/yaml.v2"
+)
+
+// DistributionFormat is for unmarshiling a email distributionList file
+type DistributionFormat struct {
+	Severity SeverityDistribution `yaml:"severity"`
+}
+
+// SeverityDistribution list of emails ordered by severity
+type SeverityDistribution struct {
+	Debug []string `yaml:"debug,omitempty"`
+	Info  []string `yaml:"info,omitempty"`
+	Warn  []string `yaml:"warn,omitempty"`
+	Error []string `yaml:"error,omitempty"`
+	Fatal []string `yaml:"fatal,omitempty"`
+}
+
+// ReadDistributionFormatYAMLFromFile loads DistributionFormat data from a YAML file
+func ReadDistributionFormatYAMLFromFile(path string) (format DistributionFormat, err error) {
+	if data, err := ioutil.ReadFile(path); err == nil {
+		err = yaml.Unmarshal(data, &format)
+	}
+	return
```

Comment:
I think I've been doing this in all my functions, as I think it actually improves readability (you know what's being returned by the function signature, and you don't have to worry that you're typing the right thing every time), but I'm open to either approach.  Generally, I've been prefering named return values over unnamed ones.